libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -590,7 +590,8 @@ bool ASTJsonConverter::visit(InlineAssembly const& _node)
 	setJsonNode(_node, "InlineAssembly", {
 		make_pair("AST", Json::Value(yul::AsmJsonConverter(sourceIndexFromLocation(_node.location()))(_node.operations()))),
 		make_pair("externalReferences", std::move(externalReferencesJson)),
-		make_pair("evmVersion", dynamic_cast<solidity::yul::EVMDialect const&>(_node.dialect()).evmVersion().name())
+		make_pair("evmVersion", dynamic_cast<solidity::yul::EVMDialect const&>(_node.dialect()).evmVersion().name()),
+		make_pair("memorySafe", _node.annotation().memorySafe)
```

Comment:
Actually, I need to check, if I can be sure that this always has been correctly assigned before exporting.